### ISSUE TITLE
devstack kilo version deprecated

### ISSUE THREAD

**Issue by jiangyurong609** [cmusatyalab/elijah-openstack#6]
currently installing devstack kilo version always failed due to many discontinuity issues. Do you have a version of devstack kilo version (with glance, nova, cinder, etc) package available?

**Comment by teiszler** [comment]
@jiangyurong609 

You are correct. Because the stable/kilo branch of OpenStack has been marked for end of life, the stackrc file in the DevStack repo points to non-existent branches for all of the OpenStack components. To remedy this, you can edit stackrc prior to running ./stack.sh in the following way:

For each of the _BRANCH variables listed in the OpenStack Server Components section, change
**-stable/kilo** 
to 
**-tags/kilo-eol**

Alternatively you can start with the sample stackrc file which I placed in the samples directory of this repository ( https://github.com/cmusatyalab/elijah-openstack/blob/master/samples/stackrc).

Ideally the stackrc in the DevStack repo (https://github.com/openstack-dev/devstack/blob/stable/kilo/stackrc) should be updated to reflect this change.
